sci-libs/libsigrokdecode/libsigrokdecode-9999.ebuild

```diff
@@ -1,43 +1,45 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI=7
 
 PYTHON_COMPAT=( python3_{6,7} )
+inherit python-single-r1
 
-inherit eutils ltprune python-single-r1
-
-if [[ ${PV} == "9999" ]]; then
+if [[ ${PV} == *9999* ]]; then
 	EGIT_REPO_URI="git://sigrok.org/${PN}"
 	inherit git-r3 autotools
 else
 	SRC_URI="https://sigrok.org/download/source/${PN}/${P}.tar.gz"
 	KEYWORDS="~amd64 ~x86"
 fi
 
-DESCRIPTION="provide (streaming) protocol decoding functionality"
+DESCRIPTION="Provide (streaming) protocol decoding functionality"
 HOMEPAGE="https://sigrok.org/wiki/Libsigrokdecode"
 
 LICENSE="GPL-3"
 SLOT="0/9999"
 IUSE="static-libs"
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
-RDEPEND=">=dev-libs/glib-2.34.0
-	${PYTHON_DEPS}"
-DEPEND="${RDEPEND}
-	virtual/pkgconfig"
+RDEPEND="${PYTHON_DEPS}
+	>=dev-libs/glib-2.34.0
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+"
 
 src_prepare() {
-	[[ ${PV} == "9999" ]] && eautoreconf
+	default
 
 	# Only a test program (not installed, and not used by src_test)
 	# is used by libsigrok, so disable it to avoid the compile.
 	sed -i \
```

Comment:
This sed fails for the live ebuild, because eautoreconf is now at the end of the function.